Containers/mastercontainer/apache.conf

```diff
@@ -0,0 +1,106 @@
+StartServers             3
+MinSpareThreads         75
+MaxSpareThreads        250
+ThreadsPerChild         25
+MaxRequestWorkers      400
+MaxConnectionsPerChild   0
+PidFile "/run/apache2/httpd.pid"
+
+ServerTokens Prod
+ServerSignature On
+
+LoadModule mpm_event_module /usr/lib/apache2/mod_mpm_event.so
+LoadModule authz_core_module /usr/lib/apache2/mod_authz_core.so
+LoadModule access_compat_module /usr/lib/apache2/mod_access_compat.so
+LoadModule log_config_module /usr/lib/apache2/mod_log_config.so
+LoadModule unixd_module /usr/lib/apache2/mod_unixd.so
+LoadModule rewrite_module /usr/lib/apache2/mod_rewrite.so
+LoadModule proxy_module /usr/lib/apache2/mod_proxy.so
+LoadModule proxy_fcgi_module /usr/lib/apache2/mod_proxy_fcgi.so
+
+User www-data
+Group www-data
+
+<Directory />
+    AllowOverride none
+    Require all denied
+</Directory>
+
+<IfModule dir_module>
+    DirectoryIndex index.html
+</IfModule>
+
+<Files ".ht*">
+    Require all denied
+</Files>
+
+ErrorLog /proc/self/fd/2
+
+LogLevel error
+
+<IfModule log_config_module>
+    LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
+    LogFormat "%h %l %u %t \"%r\" %>s %b" common
+
+    <IfModule logio_module>
+      LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\" %I %O" combinedio
```

Comment:
can we use the same logformat like below?

Reply:
just change it, how you need it